tests/daily_scraper/test_daily_scraper.py

```diff
@@ -6,309 +6,132 @@
-    {'args': {'platform': 'win32',
-              'branch': 'mozilla-central',
-              'date': '2013-07-02',
-              'build_number': 1},
-    'filename': '2013-07-02-03-12-13-mozilla-central-firefox-27.0a1.en-US.win32.installer.exe',
-    'url': 'firefox/nightly/2013/07/2013-07-02-03-12-13-mozilla-central/firefox-27.0a1.en-US.win32.installer.exe'
-    },
-
-    # -p win32 --branch=mozilla-central --stub (old format)
-    {'args': {'platform': 'win32',
-              'branch': 'mozilla-central',
-              'date': '2013-09-30',
-              'is_stub_installer': True},
-    'filename': '2013-09-30-03-02-04-mozilla-central-firefox-27.0a1.en-US.win32.installer-stub.exe',
-    'url': 'firefox/nightly/2013/09/2013-09-30-03-02-04-mozilla-central/firefox-27.0a1.en-US.win32.installer-stub.exe'
-    },
-
-    # -p win32 --branch=mozilla-central --stub (new format)
-    {'args': {'platform': 'win32',
-              'branch': 'mozilla-central',
-              'is_stub_installer': True},
-    'filename': '2013-10-01-03-02-04-mozilla-central-Firefox Installer.en-US.exe',
-    'url': 'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/Firefox Installer.en-US.exe'
-    },
-
-    # -p win32 --branch=mozilla-aurora
-    {'args': {'platform': 'win32',
-              'branch': 'mozilla-aurora'},
-    'filename': '2013-10-01-03-02-04-mozilla-aurora-firefox-27.0a1.en-US.win32.installer.exe',
-    'url': 'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-aurora/firefox-27.0a1.en-US.win32.installer.exe'
-    },
-
-    # -p win32 --branch=ux
-    {'args': {'platform': 'win32',
-              'branch': 'ux'},
-    'filename': '2013-10-01-03-02-04-ux-firefox-27.0a1.en-US.win32.installer.exe',
-    'url': 'firefox/nightly/2013/10/2013-10-01-03-02-04-ux/firefox-27.0a1.en-US.win32.installer.exe'
-    },
-
-    # --revision 1dbe350b57b1
-    {'args': {'platform': 'win32'},
-    'filename': '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.en-US.win32.installer.exe',
-    'url': 'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.win32.installer.exe'
-    },
+    ({'platform': 'win32'},
+     '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.en-US.win32.installer.exe',
+     'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/''firefox-27.0a1.en-US.win32.installer.exe'),
+    ({'platform': 'win32', 'branch': 'mozilla-central'},
+     '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.en-US.win32.installer.exe',
+     'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.win32.installer.exe'),
+    ({'platform': 'win64', 'branch': 'mozilla-central'},
+     '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.en-US.win64.installer.exe',
+     'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.win64.installer.exe'),
+    ({'platform': 'win64', 'branch': 'mozilla-central', 'date': '2013-09-30'},
```

Comment:
Mind moving this set of parameters down after the default central tests to line 36? Also add a comment that this test is for the `(old filename format)` which was part of the former comment.